src/sdk/namespace_cache.js

```diff
@@ -472,26 +527,39 @@ class NamespaceCache {
                         { md_conditions: params.md_conditions, cache_object_md: params.object_md });
                 }
 
-                this.stats_collector.update_namespace_cache_stats({
+                this.stats_collector.update_cache_stats({
                     bucket_name: params.bucket,
                     miss_count: 1,
                     range_op: true,
                 });
             } else {
                 upload_params.last_modified_time = (new Date(params.object_md.create_time)).getTime();
+                const start = process.hrtime.bigint();
```

Comment:
Perhaps we should move this into the upload async function so that it will not count the time sitting in the queue?

I'm not sure what we want here actually because the issue with including the queue time in the latency is that it means we account for other cache writes work multiple times. My instinct is to not include it, or maybe include it as a separate latency stat. 

WDYT?

Reply:
Yes, I moved it under the async, it makes more sense to me